src/gui/MixerView.cpp

```diff
@@ -68,123 +68,116 @@ MixerView::MixerView() :
-	//pal.setColor( QPalette::Window, QColor( 72, 76, 88 ) );
-	//setPalette( pal );
+	//pal.setColor(QPalette::Window, QColor(72, 76, 88));
+	//setPalette(pal);
 
-	setAutoFillBackground( true );
-	setSizePolicy( QSizePolicy::Preferred, QSizePolicy::Fixed );
+	setAutoFillBackground(true);
+	setSizePolicy(QSizePolicy::Preferred, QSizePolicy::Fixed);
 
-	setWindowTitle( tr( "Mixer" ) );
-	setWindowIcon( embed::getIconPixmap( "mixer" ) );
+	setWindowTitle(tr("Mixer"));
+	setWindowIcon(embed::getIconPixmap("mixer"));
 
 	// main-layout
-	auto ml = new QHBoxLayout;
+	auto ml = new QHBoxLayout{this};
 
 	// Set margins
-	ml->setContentsMargins( 0, 4, 0, 0 );
+	ml->setContentsMargins(0, 4, 0, 0);
 
 	// Channel area
 	m_channelAreaWidget = new QWidget;
-	chLayout = new QHBoxLayout( m_channelAreaWidget );
-	chLayout->setSizeConstraint( QLayout::SetMinimumSize );
-	chLayout->setSpacing( 0 );
+	chLayout = new QHBoxLayout(m_channelAreaWidget);
+	chLayout->setSizeConstraint(QLayout::SetMinimumSize);
+	chLayout->setSpacing(0);
 	chLayout->setContentsMargins(0, 0, 0, 0);
 	m_channelAreaWidget->setLayout(chLayout);
 
 	// create rack layout before creating the first channel
 	m_racksWidget = new QWidget;
-	m_racksLayout = new QStackedLayout( m_racksWidget );
-	m_racksLayout->setContentsMargins( 0, 0, 0, 0 );
-	m_racksWidget->setLayout( m_racksLayout );
+	m_racksLayout = new QStackedLayout(m_racksWidget);
+	m_racksLayout->setContentsMargins(0, 0, 0, 0);
+	m_racksWidget->setLayout(m_racksLayout);
 
 	// add master channel
-	m_mixerChannelViews.resize( m->numChannels() );
-	m_mixerChannelViews[0] = new MixerChannelView( this, this, 0 );
+	m_mixerChannelViews.resize(m->numChannels());
+	m_mixerChannelViews[0] = new MixerChannelView(this, this, 0);
 
-	m_racksLayout->addWidget( m_mixerChannelViews[0]->m_rackView );
+	m_racksLayout->addWidget(m_mixerChannelViews[0]->m_effectRackView);
 
 	MixerChannelView * masterView = m_mixerChannelViews[0];
-	ml->addWidget( masterView->m_mixerLine, 0, Qt::AlignTop );
+	ml->addWidget(masterView, 0, Qt::AlignTop);
 
-	QSize mixerLineSize = masterView->m_mixerLine->size();
+	auto mixerChannelSize = masterView->sizeHint();
 
 	// add mixer channels
-	for( int i = 1; i < m_mixerChannelViews.size(); ++i )
+	for (int i = 1; i < m_mixerChannelViews.size(); ++i)
 	{
 		m_mixerChannelViews[i] = new MixerChannelView(m_channelAreaWidget, this, i);
-		chLayout->addWidget( m_mixerChannelViews[i]->m_mixerLine );
+		chLayout->addWidget(m_mixerChannelViews[i]);
 	}
 
 	// add the scrolling section to the main layout
 	// class solely for scroll area to pass key presses down
 	class ChannelArea : public QScrollArea
 	{
 		public:
-			ChannelArea( QWidget * parent, MixerView * mv ) :
-				QScrollArea( parent ), m_mv( mv ) {}
+			ChannelArea(QWidget * parent, MixerView * mv) :
+				QScrollArea(parent), m_mv(mv) {}
 			~ChannelArea() override = default;
-			void keyPressEvent( QKeyEvent * e ) override
+			void keyPressEvent(QKeyEvent * e) override
```

Comment:
Extra space between types and `*`

Reply:
Fixed with commit 92878ce4030. Using reference/pointer at type like in other places in the code.